eng/test-job.yml

```diff
@@ -323,6 +323,26 @@ jobs:
           - forcerelocs
           - gcstress15
 
+    # Zip tests
+    - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+      - script: pushd $(Build.SourcesDirectory)/bin/tests/$(osGroup).$(archType).$(buildConfigUpper) && tar -czvf $(Build.SourcesDirectory)/bin/tests/$(osGroup).$(archType).$(buildConfigUpper).tar.gz ./* && popd
```

Comment:
Have you tried `ArchiveFiles@2` task (https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/archive-files?view=azure-devops) ?

Reply:
Pushed with the change.